uint8 priority
---
bool success